corehq/apps/accounting/forms.py

```diff
@@ -1743,15 +1746,39 @@ def amount(self):
                                   % adjustment_type)
 
     def adjust_balance(self, web_user=None):
-        CreditLine.add_credit(
-            -self.amount,
-            account=self.invoice.subscription.account,
-            subscription=self.invoice.subscription,
-            note=self.cleaned_data['note'],
-            invoice=self.invoice,
-            reason=self.cleaned_data['method'],
-            web_user=web_user,
-        )
+        method = self.cleaned_data['method']
+        kwargs = {
+            'account': self.invoice.subscription.account,
+            'note': self.cleaned_data['note'],
+            'reason': method,
+            'subscription': self.invoice.subscription,
+            'web_user': web_user,
+        }
+        if method == CreditAdjustmentReason.MANUAL:
+            CreditLine.add_credit(
+                -self.amount,
```

Comment:
Can't we make a `CreditLine.add_to_invoice` method to do this instead? Or is subtracting from the credit line then adding back to the credit line actually what we want to be doing (to create 2 notes or transactions or something). 
I'm not a huge fan of "this method has a side effect I don't want, so I'll just undo it later"